### PR TITLE
Add a new blog 0422

### PR DESCRIPTION
**Pull request by booooodv**

### CODE REVIEW

**Comment by booooodv** [comment]
@husiyu PTAL

File: tiflash-is-getting-faster.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+**对于这样的问题，近期我们推出了 Super Batch 优化，当开启优化时，TiDB 将会把所有需要发送到同一个 TiFlash 的请求合并，而 TiFlash 则会在内部自行进行 Raft 相关的读取容错。**
+
+通过这样的方式，TiFlash 不再对 Region 大小敏感。如下是 ontime 数据集的测试对比。
+
+![](media/tiflash-is-getting-faster/5-cop-super-batch.png)
+
+**如上测试结果可以看出，多数查询有接近一倍的提速，而这只是在较小数据量下（10 亿规模以内）的结果，如果数据量进一步增加，加速效果将更为显著。**
+
+## Join 加速
```
Comment (husiyu) [inline tiflash-is-getting-faster.md:64]: JION 加速

File: tiflash-is-getting-faster.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+TiFlash 诚然本质是依靠列存加速，但它也借助了 ClickHouse 计算层的优异实现，因此它也不仅仅是列存。TiFlash 与 TiKV 一样，拥有协处理机制。简单来说，协处理器就是替 TiDB 分担计算的机制。下面我们看下这个例子：
+
+```sql
+SELECT COUNT(*) FROM LINEORDER;
+```
+
+![](media/tiflash-is-getting-faster/1-count-order-lineorder.png)
+
+看这样一个简单的 `count` 计算的执行计划，其中 operator info 栏目中 `count(1)` 的被标记为 cop[tiflash]，这表示 TiFlash 将会执行 Hash 聚合计算 `count(1)`，而实际需要返回给 TiDB 的数据，仅仅是聚合完之后的结果，在大多数场景下，返回的数据将会非常少。这种协处理器机制，将会由各个 TiFlash 按照 Region（数据分片）为单位分布式执行。由于 TiFlash 配备了优异的计算模块，因此这部分下推优化是 TiFlash 加速的关键因素之一。
```
[the remark this answers is not in the document]
Comment (booooodv) [reply tiflash-is-getting-faster.md]: Done

File: tiflash-is-getting-faster.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+```sql
+SELECT COUNT(*) FROM LINEORDER WHERE DATE_FORMAT(LO_ORDERDATE, “%Y”) >= ‘1998’;
+```
+
+![](media/tiflash-is-getting-faster/2-count-order-date.png)
+
+![](media/tiflash-is-getting-faster/3-order-result.png)
+
+上面的执行计划中，TiFlash 只承担 TableFullScan 也就是扫表部分，而 `count(1)` 却并没有在 TiFlash 中执行。这是为何？其实原因也很简单：因为暂时 `date_format` 函数在 TiFlash 中并没有实现，因此从谓词过滤以及所有之后的计算都将无法加速。这也许会带来几倍甚至十几倍的速度差距。
```
[the remark this answers is not in the document]
Comment (booooodv) [reply tiflash-is-getting-faster.md]: Done

**Comment by husiyu** [comment]
LGTM

File: tiflash-is-getting-faster.md (hunk 1 above)
Comment (booooodv) [reply tiflash-is-getting-faster.md]: Done

**Comment by YiniXu9506** [comment]
LGTM

**Pull request merged**